testing/downloadSpark.sh

```diff
@@ -26,10 +26,10 @@ fi
 SPARK_VERSION="${1}"
 HADOOP_VERSION="${2}"
 
-echo ${SPARK_VERSION} | grep "^1.[123].[0-9]" > /dev/null
-if [ $? -eq 0 ]; then
+echo "${SPARK_VERSION}" | grep "^1.[123].[0-9]" > /dev/null
+if [[ "$?" -eq 0 ]]; then
```

Comment:
I see what you mean. But from the [syleguide on quoting](https://google.github.io/styleguide/shell.xml?showone=Quoting#Quoting)

>Never quote literal integers.